_dev/js/back/src/components/panel/payment-mode.vue

```diff
@@ -114,18 +160,27 @@
         this.$store.dispatch('updatePaymentMode', mode).then(() => {
           this.$store.dispatch('getOnboardingLink');
         });
+      },
+      cancelCaptureMode() {
+        var selector = "#intent-mode input[value='" + this.captureMode + "']";
+        document.querySelector(selector).click();
+      },
+      updateCaptureMode() {
+        var captureMode =
+          this.captureMode === 'CAPTURE'
+            ? 'AUTHORIZE'
+            : this.captureMode === 'AUTHORIZE'
+            ? 'CAPTURE'
+            : '';
+        if (captureMode !== '') {
+          this.$store.dispatch('updateCaptureMode', captureMode);
+        }
       }
     },
     computed: {
       captureMode: {
         get() {
           return this.$store.state.configuration.captureMode;
```

Comment:
probably this would be nicer if refactored as a [Vuex getter](https://vuex.vuejs.org/fr/guide/getters.html) and then mapped using `mapGetters`